recipes/BinauralWSJ0Mix/separation/hparams/convtasnet-parallel.yaml

```diff
@@ -0,0 +1,183 @@
+# ################################
+# Model: Convtasnet for binaural source separation
+# https://ieeexplore.ieee.org/abstract/document/9053215
+# Dataset : Binaural WSJ0-2mix and WSJ0-3mix
+# ################################
+# Basic parameters
+# Seed needs to be set at top of yaml, before objects with parameters are made
+#
+seed: 1234
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+
+# Data params
+
+# Output directory for writing binaural wsj0-2mix.
+data_folder: !PLACEHOLDER
+# Path to the folder containing wsj0/
+wsj_root: !PLACEHOLDER
+data_freqs: ['8k'] # or if you want the whole dataset ['8k','16k']
+data_modes: ['min'] # or if you want the whole dataset ['min', 'max']
+
+# the path for wsj0/si_tr_s/ folder -- only needed if dynamic mixing is used
+# e.g. /yourpath/wsj0-processed/si_tr_s/
+base_folder_dm: !ref <wsj_root>/wsj0/si_tr_s/
+
+# the path for binaural-wsj0mix datasets generation automatically
+datasets_generation: !ref <data_folder>/Binaural-WSJ0Mix-main
+hrtf_wav_path: !ref <datasets_generation>/CIPIC_hrtf_database/wav_database/
+
+experiment_name: convtasnet-parallel
+output_folder: !ref results/<experiment_name>/<seed>
+train_log: !ref <output_folder>/train_log.txt
+save_folder: !ref <output_folder>/save
+train_data: !ref <save_folder>/binaural_wsj0-2mix_tr.csv
+valid_data: !ref <save_folder>/binaural_wsj0-2mix_cv.csv
+test_data: !ref <save_folder>/binaural_wsj0-2mix_tt.csv
+skip_prep: False
+
+
+# Experiment params
+auto_mix_prec: False # Set it to True for mixed precision
+test_only: False
+num_spks: 2 # set to 3 for wsj0-3mix
+progressbar: True
+save_audio: True # Save estimated sources on disk
```

Comment:
Yes, it's okay. I have set the save_audio true in three .yamls and saved 10 items in the inference. These results are in the save/audio_results folder that I shared with you through Google driver. Thanks.

Reply:
Great, thanks!